mlflow/sagemaker/container/__init__.py

```diff
@@ -61,9 +69,13 @@ def _serve():
     if pyfunc.FLAVOR_NAME not in m.flavors:
         raise Exception("Only supports pyfunc models and this is not one.")
     conf = m.flavors[pyfunc.FLAVOR_NAME]
+    model_py_version = None
+    if pyfunc.PY_VERSION in conf:
+        model_py_version = conf[pyfunc.PY_VERSION]
+
     bash_cmds = []
     if pyfunc.ENV in conf:
-        print("activating custom environment")
+        print_flush("activating custom Anaconda environment")
```

Comment:
If this is a concern, should we consider setting PYTHONUNBUFFERED so all prints are flushed?